lib/scanner/index.js

```diff
@@ -100,13 +100,17 @@ class Scanner extends Emitter {
    *
    * We don't fire the callback until all interactions
    * (touches/animations) are complete to prevent
-   * re-rendering and droppping frames.
+   * re-rendering and dropping frames.
    */
-  onItemFound({ url, distance }) {
+  onItemFound({ url, distance, latitude, longitude }) {
```

Comment:
Should we use default values for latitutude and longitude to null, not all items found will have this attributes?